README.md

```diff
@@ -10,8 +10,8 @@ Aerleon is a fork of [Capirca](https://github.com/google/capirca) with the follo
 - Support for new firewall platforms can be added through plugins. Plugins for
   common platforms are built-in. Users with experimental or non-public platforms
   can add support without forking this repo.
-- Policy files can be given as YAML (.pol.yaml).
-- Address Book data can be loaded from standard data formats like JSON, CSV.
+- Policy files can be given as (.pol.yaml).
```

Comment:
Accidentally a word?

Reply:
Will provide suggestions, the extension no longer needs to be `.pol.yaml`